ansible/ansible_collections/agentic/agent_configs/plugins/modules/codex_cli_hook.py

```
@@ -1,9 +1,34 @@
+
+    - name: Install a project Codex Stop hook
+      agentic.agent_configs.codex_cli_hook:
+        agent_executable: /home/payton/.local/bin/codex
+        scope: project
+        project_dir: /srv/my-repo
+        event: Stop
+        command: /srv/my-repo/.codex/hooks/stop.sh
+        timeout: 30
+"""
+
 #!/usr/bin/python
```

Ruff (0.15.11)

[warning] Shebang should be at the beginning of the file

(EXE005)